src/main/java/net/mcreator/ui/blockly/BlocklyEditorType.java

```diff
@@ -22,7 +22,8 @@
 public enum BlocklyEditorType {
 
 	PROCEDURE("procedures", "ptpl", "event_trigger"),
-	AI_TASK("ai_setup", "aitpl", "aitasks_container");
+	AI_TASK("ai_setup", "aitpl", "aitasks_container"),
+	FEATURE("features", "ftpl", "feature_container");

```

Comment:
While I do plan to add feature templates, I'd prefer to keep the first PR as small as possible. I'd revert these changes and just use "feature_container" for now

Reply:
It was only to avoid more changes in the future and make the code already similar to everything else, so it has some consistency.